db/schema.rb

```diff
@@ -10,11 +10,20 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_07_12_200541) do
+ActiveRecord::Schema.define(version: 2021_07_15_082312) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "games", force: :cascade do |t|
+    t.bigint "user_id", null: false
```

Comment:
I noticed that some of our other foreign keys are just regular integers. Do we want them to be `bigint` as well?